man/systemd.link.xml

```diff
@@ -496,6 +496,93 @@
           </variablelist>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Advertise=</varname></term>
+        <listitem>
+          <para>The Advertise option is used to set what speeds and modes of operation
+          (duplex mode) are advertised for auto-negotiation. The
```

Comment:
"This sets what speed and duplex mode of operation are advertised"

Reply:
ok